pkg/controller/volume/persistentvolume/pv_controller.go

```diff
@@ -451,10 +451,12 @@ func (ctrl *PersistentVolumeController) syncBoundClaim(claim *v1.PersistentVolum
 		}
 		return nil
 	}
-	obj, found, err := ctrl.volumes.store.GetByKey(claim.Spec.VolumeName)
+	obj, found, err := ctrl.volumes.store.GetByKey(claimToVolumeKey(claim))
 	if err != nil {
 		return err
 	}
+	klog.Infof("======= %s", claimToVolumeKey(claim))
```

Comment:
remove these?

Reply:
will do. this PR is still WIP.

Reply:
manual e2e is working. however the changes here have disturbed lots of unit tests and probably int tests. fixing them.

Reply:
this is removed